### DISCUSSION TITLE
intro silence option

### DISCUSSION THREAD

**Discussion by zero9612** [ppy/osu#25849] [Ideas] [answered]
there is currently no way of stopping the loud welcome to osu! on lazer that plays on startup, adding a way to do so would be very nice as i feel like the intro for lazer is incredibly loud

**Comment by cihe13375** [comment]
settings - search for "interface voices"

**Reply by zero9612** [reply]
that actually didnt solve it, it still yells welcome to osu at system volume for me

**Reply by peppy** [reply, marked as answer]
That might depend on which intro you have selected. You can also choose this in settings (search "intro"). There's also a setting to opt out of startup music (search "music").
